src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
```

Comment:
```suggestion
  const inversedRobot = {};
```
Avoid using new word in the variable naming, it's a kind of confusing and don't bring any meaning whatsoever

Reply:
Done!